### PR TITLE
Update ovirt infra trac

### PR DESCRIPTION
**Pull request by arielb2**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-  Update the link for the new ovirt infra trac

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/README.md#how-to-contribute): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/develop/infra/becoming-an-infrastructure-team-member.html.md [hunk 1]
```diff
@@ -13,14 +13,14 @@ This page describes the process for becoming a member of the Infrastructure team
 The Infrastructure team primarily uses three resources to get stuff done:
 
 *   [The infra mailing list](http://lists.ovirt.org/mailman/listinfo/infra) for asynchronous communication
-*   [oVirt infra trac](https://fedorahosted.org/ovirt/report/1) for issue tracking
+*   [oVirt infra trac](https://ovirt-jira.atlassian.net/projects/OVIRT/issues) for issue tracking
```
Comment (mykaul) [inline source/develop/infra/becoming-an-infrastructure-team-member.html.md]: It's JIRA now, not trac.

File: source/develop/infra/infrastructure-team-meetings.html.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ This page is for tracking agenda, notes, logs, and etc. about each oVirt Infrast
 *   *When:* Monday @ 15:00 UTC (may change during DST changes) - always at 10:00am US Eastern.
     -   To see in your timezone date -d 'MONDAY 1000 EST'
 *   *Where:* On IRC: #ovirt on irc.oftc.net
-    -   The Trac issue tracker used bu the infrastructure team is [here](https://fedorahosted.org/ovirt/report), rather than in the general Red Hat bugzilla.
+    -   The Trac issue tracker used bu the infrastructure team is [here](https://ovirt-jira.atlassian.net/projects/OVIRT/issues), rather than in the general Red Hat bugzilla.
```
Comment (mykaul) [inline source/develop/infra/infrastructure-team-meetings.html.md]: Same here - JIRA, not Trac.

**Comment by arielb2** [comment]
done.
thanks for your comments.

**Comment by gregsheremeta** [comment]
out of date, closing

**Pull request closed without merging**